docs/Tutorial_DAMP.ipynb

```diff
@@ -0,0 +1,778 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #93.        return pruned
</pre><p>There's no point in returning anything since <code>pruned</code> is a reusable array that is updated inplace</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/TDAmeritrade/stumpy/pull/872/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Reply:
<p>Right! I will fix it! I will just do bare <code>return</code>  <span class="ql-cursor">﻿</span></p>